tests/Spec/Nri/Ui/Tabs/V4.elm

```diff
@@ -0,0 +1,36 @@
+module Spec.Nri.Ui.Tabs.V4 exposing (all)
+
+import Accessibility.Styled as Html
+import Expect
+import List.Zipper.Extra
+import Nri.Ui.Tabs.V4 as Tabs
+import ProgramTest
+import Test exposing (..)
+
+
+all : Test
+all =
+    describe "Nri.Ui.Tabs.V4"
+        [ test "works with ProgramTest.clickButton" <|
+            \() ->
+                ProgramTest.createSandbox
+                    { init = Err "No msg"
+                    , update = \newResult _ -> newResult
+                    , view =
+                        \_ ->
+                            Tabs.view
+                                { title = Nothing
+                                , onSelect = Ok
+                                , tabs =
+                                    List.Zipper.Extra.from []
+                                        (Tabs.Tab "First tab" "ID_FIRST")
+                                        [ Tabs.Tab "Second tab" "ID_SECOND" ]
+                                , content = \_ -> Html.text ""
+                                , alignment = Tabs.Center
+                                }
+                                |> Html.toUnstyled
+                    }
+                    |> ProgramTest.start ()
+                    |> ProgramTest.clickButton "Second tab"
```

Comment:
Before the change in this PR to use `Html.button`, this line would fail to find the tab.